app/javascript/dashboard/App.vue

```diff
@@ -6,6 +6,8 @@
     :class="{ 'app-rtl--wrapper': isRTLView }"
   >
     <update-banner :latest-chatwoot-version="latestChatwootVersion" />
+    <payment-pending-banner />
+    <upgrade-banner />
```

Comment:
Let us render this component only after account information is available. See the date calculation.
<img width="1434" alt="Screenshot 2023-06-09 at 5 16 05 PM" src="https://github.com/chatwoot/chatwoot/assets/2246121/64952a75-d0b4-484f-b143-3d82291d07e2">